apps/fishing-map/features/map/Map.tsx

```diff
@@ -23,6 +26,8 @@ import {
 } from 'routes/routes.selectors'
 import { AsyncReducerStatus } from 'utils/async-slice'
 
+import { Logo } from '../../../../libs/ui-components/src/logo/Logo'
```

Comment:
This is going to fail, please use the `@globalfishingwatch/ui-components` import